platform.live-iso.ipp.yaml

```diff
@@ -0,0 +1,217 @@
+                target: /tmp-squashfs-dir/boot
+      - type: org.osbuild.squashfs
+        inputs:
+          tree:
+            type: org.osbuild.tree
+            origin: org.osbuild.pipeline
+            references:
+              # XXX: we should rename this pipeline now that we don't use it as
+              # a buildroot
+              - name:build
+        devices:
+          metal:
+            type: org.osbuild.loopback
+            options:
+              filename: metal.raw
+              partscan: true
+              read-only: true
+              sector-size: 512
+        mounts:
+              - name: root
+                type: org.osbuild.xfs
+                source: metal
+                partition:
+                  mpp-format-int: '{image.layout[''root''].partnum}'
+                target: /tmp-squashfs-dir
+              - name: boot
+                type: org.osbuild.ext4
+                source: metal
+                partition:
+                  mpp-format-int: '{image.layout[''boot''].partnum}'
+                target: /tmp-squashfs-dir/boot
+              - name: efi
+                type: org.osbuild.fat
+                source: metal
+                partition:
+                  mpp-format-int: '{image.layout[''EFI-SYSTEM''].partnum}'
+                target: /tmp-squashfs-dir/boot/efi
+        options:
+          filename: initrd-rootfs/root.squashfs
+          coreos: true
+          compression:
+            method: "lz4"
+      - type: org.osbuild.coreos.live-iso
+        inputs:
+          tree:
+            type: org.osbuild.tree
+            origin: org.osbuild.pipeline
+            references:
+              # XXX: we should rename this pipeline now that we don't use it as
+              # a buildroot
+              - name:build
+        devices:
+          metal:
+            type: org.osbuild.loopback
+            options:
+              filename: metal.raw
+              partscan: true
+              read-only: true
+              sector-size: 512
+          metal4k:
+            type: org.osbuild.loopback
+            options:
+              filename: metal4k.raw
+              partscan: true
+              read-only: true
+              sector-size: 4096
+          efiboot_img:
+            type: org.osbuild.loopback
+            options:
+              filename: efiboot.img
+        mounts:
+              - name: root
+                type: org.osbuild.xfs
+                source: metal
+                partition:
+                  mpp-format-int: '{image.layout[''root''].partnum}'
+                target: /tmp-squashfs-dir
+              - name: boot
+                type: org.osbuild.ext4
+                source: metal
+                partition:
+                  mpp-format-int: '{image.layout[''boot''].partnum}'
+                target: /tmp-squashfs-dir/boot
+        options:
+          efiboot_img_filename:
+            mpp-format-string: 'efiboot.img'
+      - type: org.osbuild.coreos.osmet
```

Comment:
definitely a bit weird to have this stage called twice? why do we need a minimal-iso?